lib/assessment.js

```diff
@@ -5,6 +5,12 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  var result = argument[0];
+    for(var i =1; i < argument.legth; i++) {
```

Comment:
There's a typo in the condition that will make this function fail in subtle and strange ways.